pkg/rust/binary/parse.go

```diff
@@ -46,7 +46,9 @@ func (p *Parser) Parse(r dio.ReadSeekerAt) ([]types.Library, []types.Dependency,
 	var deps []types.Dependency
 	for _, pkg := range info.Packages {
 		if pkg.Kind == rustaudit.Runtime {
+			pkgID := utils.PackageID(pkg.Name, pkg.Version)
 			libs = append(libs, types.Library{
+				ID:       pkgID,
```

Comment:
Is the problem that `ID` is missing in `types.Library`?

Reply:
yes, ID is needed

Reply:
all packages have an empty string instead of ID now.

Reply:
OK